sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -2831,6 +2833,35 @@ def __GetBodiesFromQueryResult(result):
 
             return __GetBodiesFromQueryResult(result)
 
+    def _GetQueryPlanThroughGateway(self, query, resource_link):
+        supported_query_features = (documents._QueryFeature.Aggregate + "," +
+                                    documents._QueryFeature.CompositeAggregate + "," +
+                                    documents._QueryFeature.Distinct + "," +
+                                    documents._QueryFeature.MultipleOrderBy + "," +
```

Comment:
You can only add these if the SDK actually supports them. 

So far I think python is just 

* Aggregate
* ORDER BY
* TOP

Reply:
Can you also make sure that Java and JS did this right? I think I might have missed it.

Reply:
Oh I see. You added DISTINCT + OFFSET + LIMIT support in this PR.